lib/cc/cli/engines.rb

```diff
@@ -9,6 +9,12 @@ module Engines
       autoload :Install, "cc/cli/engines/install"
       autoload :List, "cc/cli/engines/list"
       autoload :Remove, "cc/cli/engines/remove"
+
+      REGISTRY_LIST ||= CC::Analyzer::EngineRegistry.new.list
```

Comment:
Running code when defining a constant can cause some weird things (related to the way that classes are loaded). Can you move this to a method? Alternatively, what do you think of being able to call `CC::Analyzer::EngineRegistry.exists?(engine_name)`?